packages/design-system/src/styles/components/_Table.scss

```diff
@@ -105,102 +104,86 @@ $table-dense-padding: $spacer-half $spacer-2 !default;
     display: block;
   }
 
-  td,
-  th {
-    border: 0;
-    border-top: 1px solid;
-
-    /* autoprefixer grid: on */
-    display: grid;
-    // CSS grid doesn't support word-wrap. Use minmax as a workaround
-    // https://css-tricks.com/css-grid-in-ie-debunking-common-ie-grid-misconceptions/#fit-content-is-not-ie-friendly-but
-    grid-template-columns: minmax(0, $table-stacked-header-max-width) 1fr;
-    position: relative;
-
-    // The psuedo element content is generated to display the row header `data-title` for the Tablecell content
-    /* stylelint-disable-next-line selector-no-qualifying-type */
-    &[data-title]::before {
-      content: attr(data-title);
-      font-weight: bold;
-
-      /* autoprefixer grid: on */
-      grid-column: 1;
-      max-width: $table-stacked-header-max-width;
-      width: calc(100% - #{$spacer-2});
-    }
-
-    // TableCell content is wrapped in a span for IE11 grid support
-    > span {
-      // Fixes IE11 display bug https://css-tricks.com/css-grid-in-ie-debunking-common-ie-grid-misconceptions/
-      display: block;
+  thead {
+    display: none;
+  }
 
-      /* autoprefixer grid: on */
-      grid-column: 2;
-      width: fit-content;
-    }
+  th,
+  td {
+    border-bottom-width: 0;
+    display: block;
   }
 
   tr {
-    border: 1px solid;
+    border-bottom-color: $table-border-color;
+    border-bottom-style: solid;
+    border-bottom-width: $spacer-half;
+
+    border-top-width: 0;
     display: block;
-    margin: 0 0 1rem;
-    padding-bottom: 2px;
-    position: relative;
 
-    &::before {
-      background-color: $color-gray-lightest;
-      border-right: 1px solid;
-      bottom: 0;
-      content: '';
-      display: block;
-      left: 0;
-      max-width: $table-stacked-header-max-width;
-      position: absolute;
-      top: 0;
-      width: 100%;
+    th:first-child,
+    td:first-child {
+      border-top-width: 0;
+    }
+
+    &:nth-child(odd) {
+      td,
+      th {
+        background-color: inherit;
```

Comment:
What is this for?

Reply:
Thanks for this pick up, I've removed the striped styles from stacked table initially, but have now updated to apply striped styles on stacked table as well. 
Also, for our styles, the gray stripe is applied on even rows whereas uswds applied stripes on odd rows. Do you think we should change to 'odd' rows as well?

Reply:
I saw that the USWDS stacked table design doesnt use the striped styles, so I'm actually included to do the same and removed striped styles from stacked tables

Reply:
and as per our convo, we should leave the striped update to a separate ticket and PR